py/desispec/tile_qa_plot.py

```diff
@@ -29,11 +32,13 @@
 from desiutil.dust import ebv as dust_ebv
 from astropy import units
 from astropy.coordinates import SkyCoord
+from datetime import datetime
+
 
 log = get_logger()
 
 # AR tile radius in degrees
-tile_radius_deg = 1.628
+tile_radius_deg = get_tile_radius_deg()
```

Comment:
Oh, I've also made a very minor change: as the script now has a dependency on desimodel, I've replaced here the hard-coded `tile_radius_deg = 1.628` by a call to the `desimodel.footprint.get_tile_radius_deg()` function.
But that information is only used in some plotting routines for the tile-qa-TILEID-thruNIGHT.png file, so it is harmless.